_posts/documentation/2014-03-31-auto-narrate.md

```diff
@@ -9,7 +9,11 @@ bookmark: auto-narrate
   <img class="box-shadow" src="./img/docs/auto-narrate-1.png">
 </p>
 
-**Auto Narrate** creates synthesized audio from your speaker notes, and embeds it on each slide. It's great for figuring out exactly what your speech will sound like, or creating pre-packaged presentations that can play automatically. If you'd like, you can go one step further and easily record your own replacements for the generated audio.
+**Auto Narrate** creates synthesized audio from your speaker notes, and embeds it on each slide. It's great for figuring out exactly what your speech will sound like, or creating pre-packaged presentations that can play automatically. If you'd like, you can go one step further and easily record your own replacements for the generated audio. In the following video, the animation was created by using Auto Narrate.
+
```

Comment:
`animation was created` is misleading. Don't you mean, the `audio narration was created`?